packages/circuits/circuits/proveReputation.circom

```diff
@@ -128,7 +134,7 @@ template ProveReputation(STATE_TREE_DEPTH, EPOCH_KEY_NONCE_PER_EPOCH, SUM_FIELD_
     /* 3. Check if user has reputation less than max_rep */
     // if proving max_rep > 0, check if data[1] >= data[0] + max_rep
 
-    component max_rep_check = GreaterEqThan(252);
+    component max_rep_check = GreaterEqThan(66);
```

Comment:
why a `66` here?

Reply:
The input is `data[0] + max_rep`. Both of these are 64 bits, so `2^64 + 2^64 = 2^65`. I added one more bit for safety to get 66.